Makefile

```diff
@@ -47,6 +47,13 @@ testapps-with-numpy-aab: virtualenv
     --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,urllib3,chardet,idna,sqlite3,setuptools,numpy \
     --arch=armeabi-v7a --arch=arm64-v8a --arch=x86_64 --arch=x86 --release
 
+testapps-service_library-aar: virtualenv
+	. $(ACTIVATE) && cd testapps/on_device_unit_tests/ && \
+    python setup.py aar --sdk-dir $(ANDROID_SDK_HOME) --ndk-dir $(ANDROID_NDK_HOME) \
+    --bootstrap service_library \
+    --requirements python3 \
+    --arch=arm64-v8a --release
```

Comment:
There's a reason to only run tests on `arm64-v8a`? (Or is it just for having a faster CI?)

Reply:
there was no support for multi-arch in `service_library` bootstrap. now fixed, and x86 added to test multi-arch support.